### PR TITLE
Postgres Tests

### PR DESCRIPTION
**Pull request by illmade**
github: FintanH  and I thought some instructions on the README might be handy to get the tests passing with the postgres flag set to true.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -240,3 +240,41 @@ There are many differences between SQL syntax and functions supported by differe
 - PostgreSQL: `Database.Esqueleto.PostgreSQL`
 
 In order to use these functions, you need to explicitly import their corresponding modules.
+
+
+### Tests and Postgres
+
+To ```stack test``` with Postgresql you'll need to set the Postgres flag to true in esqueleto.cabal
```
Comment (FintanH) [inline README.md]: There's another way to do this: `stack test --flag esqueleto:postgresql`

File: README.md [hunk 2]
```diff
@@ -240,3 +240,41 @@ There are many differences between SQL syntax and functions supported by differe
...
+
+To ```stack test``` with Postgresql you'll need to set the Postgres flag to true in esqueleto.cabal 
+
+```
+Flag postgresql
+  Description: test postgresql. default is to test sqlite.
+  Default: False
+```
+
+If you don't have Postgres install it. Using apt-get it's just:
```
Comment (FintanH) [inline README.md]: Add instructions for other users maybe? I'm thinking `homebrew` or else link to postgres site itself.

File: README.md [hunk 3]
```diff
@@ -240,3 +240,41 @@ There are many differences between SQL syntax and functions supported by differe
...
+```$ sudo -u postgres createuser esqutest```
+
+```$ sudo -u postgres createdb esqutest```
+
+```
+$ sudo -u postgres psql
+postgres=# \password esqutest
+```
+
+Now ```stack test``` should envoke and pass all the Postgres tests.
```
Comment (FintanH) [inline README.md]: Repeat from comment above `stack test --flag esqueleto:postgresql`

**Comment by illmade** [comment]
Added homebrew instructions and command line flag instruction

**Comment by bitemyapp** [comment]
This is great, thank you! @illmade @FintanH is this good to go?

**Comment by FintanH** [comment]
Merged 👍

**Pull request merged**